### PR TITLE
[VL] Port CI changes to branch-1.2 and pick simdjson related fix

### PR DESCRIPTION
**Pull request by PHILO-HE**
## What changes were proposed in this pull request?

For accelerating CI validation for PR targeting to branch-1.2. Also included json related fix, which is necessary to 1.2 branch.

### CODE REVIEW

**Comment by PHILO-HE** [comment]
@weiting-chen

File: .github/workflows/util/install_spark_resources.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    mv jars ${INSTALL_DIR}/shims/spark33/spark_home/assembly/target/scala-2.12 && \
+    wget -nv https://github.com/apache/spark/archive/refs/tags/v3.3.1.tar.gz && \
+    tar --strip-components=1 -xf v3.3.1.tar.gz spark-3.3.1/sql/core/src/test/resources/  && \
+    mkdir -p shims/spark33/spark_home/ && \
+    mv sql shims/spark33/spark_home/
+    ;;
+3.4)
+    # Spark-3.4
+    cd ${INSTALL_DIR} && \
+    wget -nv https://archive.apache.org/dist/spark/spark-3.4.3/spark-3.4.3-bin-hadoop3.tgz && \
```
Comment (weiting-chen) [inline .github/workflows/util/install_spark_resources.sh:53]: branch-1.2 support spark 3.4.2

Comment (PHILO-HE) [reply .github/workflows/util/install_spark_resources.sh]: @weiting-chen, just updated, thanks for your comment!

**Review by weiting-chen — APPROVED** [review]
Code looks good to me.
But testing with uniffle is failed with rss register issue, please check it out.

**Pull request merged**